dist/bin/kctf-config

```diff
@@ -176,11 +196,24 @@ function kctf_config_create {
   CONFIG_PATH="${KCTF_CTF_DIR}/kctf/config/${CONFIG_NAME}"
   shift
 
-  if [[ -z "$PROJECT" ]]; then
-    echo "Missing required argument \"--project\"." >&2
-    kctf_config_create_usage
-    return 1
-  fi
+  case "${CLUSTER_TYPE}" in
+    gce)
+      if [[ -z "$PROJECT" ]]; then
+        echo "Missing required argument \"--project\"." >&2
+        kctf_config_create_usage
+        return 1
+      fi
+      ;;
+    kind)
+      if [[ -z "$PROJECT" ]]; then
+        PROJECT="kind-project"
```

Comment:
I worry a bit about the project ending up somewhere that does a fetch (eg, gcr) can we use a project we own? Just in case.

Reply:
We could also change the code to use a different image URL, I think that's the only reason why I had to set this.

Reply:
done, it's not using the PROJECT anymore